scripts/include/config.sh

```diff
@@ -1,5 +1,3 @@
-#!/usr/bin/env bash
```

Comment:
Yeah, does nothing useful on its own - putting the extension on there let me get rid of the shebang, yay!